src/style.css

```
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--5 :nth-child(n),
+.stars--4 :not(:last-child),
```

если в будущем появятся стили для stars--6 и stars--7?
лучше описать надежные стили, которые не придется редактировать
```suggestion
.stars--5 :nth-child(-n + 5),
.stars--4 :nth-child(-n + 4),
```